deno.json

```diff
@@ -1,7 +1,9 @@
 {
   "tasks": {
     "server": "export REDIS_PORT=6666; export REDIS_URL=127.0.0.1; export REDIS_PASSWORD=12345; export PORT=8000; deno run --allow-read --allow-net --allow-env server/src/index.ts",
+    "server-repl": "export REDIS_PORT=6666; export REDIS_URL=127.0.0.1; export REDIS_PASSWORD=12345; export PORT=8000; deno",
```

Comment:
can you add a comment here or in `dev_notes.md` what this is used for?

Reply:
👍 (dev_notes)